web/src/components/layout/footer.tsx

```diff
@@ -31,9 +31,9 @@ export default React.memo(() => {
   return (
     <footer>
       <div id="help-links">
-        <LocaleLink id="help" to={URLS.FAQ}>
+        <LocaleLink to={URLS.FAQ}>
           <SupportIcon />
-          <Localized id="help" onClick={() => trackNav('help', locale)}>
+          <Localized id="faq" onClick={() => trackNav('faq', locale)}>
```

Comment:
I'm happy to change the `trackNav` value back, if you think it's important to keep consistency here for analytics. Let me know!

Reply:
I think it's fine, the link in the footer tracks `faq` as well.